webapp/components/wpt-metadata.js

```diff
@@ -202,12 +202,12 @@ class WPTMetadata extends PathInfo(LoadingState(PolymerElement)) {
     let displayedMetadata = [];
     for (const test of Object.keys(metadata).filter(k => this.shouldShowMetadata(k, path, testResultSet))) {
       const seenURLs = new Set();
```

Comment:
Optional nit: maybe rename this to `seenProductURLs`, and perhaps add a delimiter between the product and the URL in case you need to inspect it for debugging.

Reply:
Done